### ISSUE TITLE
Install issue with php-fpm

### ISSUE THREAD

**Issue by Natetronn** [phpdocker-io/phpdocker.io#354]
Any thoughts on why php-fpm is having an issue when `docker compose up` is run? Can't seem to get MySQL/MariaDB or Alpine/Nginx installed unless I comment out the php-fpm section (but then that doesn't work)

```
[+] Building 0.2s (2/2) FINISHED                                                                                                                                                            
 => [php-fpm internal] load build definition from Dockerfile                                                                                                                           0.1s
 => => transferring dockerfile: 2B                                                                                                                                                     0.0s
 => [php-fpm internal] load .dockerignore                                                                                                                                              0.1s
 => => transferring context: 2B                                                                                                                                                        0.0s
failed to solve: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount3730572753/Dockerfile: no such file or directory
```

**Comment by Natetronn** [comment]
```
###############################################################################
#                          Generated on phpdocker.io                          #
###############################################################################
version: '3.1'
services:
    mariadb:
        image: 'mariadb:10.10'
        working_dir: /application
        volumes:
            - '.:/application'
        environment:
            - MYSQL_ROOT_PASSWORD=blablabla
            - MYSQL_DATABASE=blablabla
            - MYSQL_USER=blablabla
            - MYSQL_PASSWORD=blablabla
        ports:
            - '50003:3306'

    webserver:
        image: 'nginx:alpine'
        working_dir: /application
        volumes:
            - '.:/application'
            - './phpdocker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf'
        ports:
            - '50000:80'

    php-fpm:
        build: phpdocker/php-fpm
        working_dir: /application
        volumes:
            - '.:/application'
            - './phpdocker/php-fpm/php-ini-overrides.ini:/etc/php/8.2/fpm/conf.d/99-overrides.ini'

```

**Comment by luispabon** [comment]
When you unzipped the file did you also copy the phpdocker folder alongside
the docker compose file?

**Comment by Natetronn** [comment]
Yup. Initially I get the following:

> unable to prepare context: path "phpdocker/php-fpm" not found

So I created those those folders (and the nginx one, as well as tried creating the files it's suppose to be writing to, at one point), but I still get the above error. I also tried using the full path instead of relative paths in the docker-compose.yml file and anything else I could think of in between all that, including changing permissions and such.
